assets/queries/k8s/cpu_requests/query.rego

```diff
@@ -0,0 +1,33 @@
+package Cx
```

Comment:
Check the query attached, it does have a more specific searchKey

[query.zip](https://github.com/Checkmarx/kics/files/5629863/query.zip)